### PR TITLE
Add index fairease

### PR DESCRIPTION
**Pull request by Marie59**
Draft of the index for the fairease subdomain @yvanlebras and @bgruening.

### CODE REVIEW

**Comment by bgruening** [comment]
Do we want to have this before the workshop?

**Comment by Marie59** [comment]
No we don't need it for the training I think. I think this usegalaxy will not be needed right away. There is still some debate on how to proceed here.

File: index-earth-system.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Lesson | Slides | Hands-on | Input dataset | Workflows | Galaxy History
+--- | --- | --- | --- | --- | ---
+Welcome and introduction to Galaxy | [<i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-short/slides.html){:target="_blank"} / [<i class="fa fa-video-camera" aria-hidden="true"></i>](https://training.galaxyproject.org/videos/watch.html?v=introduction/tutorials/galaxy-intro-short/slides){:target="_blank"} | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.582600){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/workflows/){:target="_blank"} |
+
+# Tools
+
+Multiple tools on Earth System, Environment and Biodiversity data analysis will be available in this custom Galaxy instance:
+
+- **General tools**
+    - QGIS
```
Comment (bgruening) [inline index-earth-system.md:59]: you could link all those tools

**Review by bgruening — APPROVED** [review]

**Review by detocjerome — APPROVED** [review]

**Comment by bgruening** [comment]
Lets get this in and you can finetune it tomorrow ....

**Comment by bgruening** [comment]
Website should be here: https://usegalaxy-eu.github.io/index-earth-system.html

**Pull request merged**